www/templates/diary/diary_list_item.html

```diff
@@ -0,0 +1,67 @@
+<div>
+    <div class="start-time-tag">{{tripgj.display_start_time}}</div>
+    <div  style="padding-left: 19%;">
+
+    <ion-item id="diary-item" style="background-color: transparent;" class="list-item">
+        <div ng-class="listExpandClass" ng-if="tripgj.common.different" id="hidden-{{tripgj.data.id}}">
+            {{tripgj.common.displayEarlierOrLater}}
+        </div>
+
```

Comment:
I am skipping this file during the review since it is obsolete.

Reply:
_**Confirmed this change as complete**_